src/scss/block-patterns/_cards.scss

```diff
@@ -6,38 +8,71 @@
 	align-items: stretch;
 	gap: var(--wp--preset--spacing--20);
 
+	&.is-layout-flow > * {
+		margin-block-start: 0;
+		margin-block-end: 0;
+	}
+
+	.ucsc__card-grid & {
+		height: 100%;
+	}
+}
+
+.ucsc__card-inner {
+	height: 100%;
+}
+
+/* Card images */
+:where(.ucsc__card .wp-block-image),
+:where(.ucsc__card .wp-block-post-featured-image) {
+
 	a {
-		color: var(--wp--preset--color--black);
-		font-weight: 700;
+		display: block;
+		overflow: hidden;
+
+		img {
+			transition: transform 0.3s ease;
+		}
 
 		&:hover,
 		&:focus {
-			color: var(--wp--preset--color--ucsc-primary-blue);
+
+			img {
+				transform: scale(1.15);
+			}
 		}
 	}
+}
 
-	&.ucsc__card-yellow {
+/* Links in card overlines and titles */
+:where(.ucsc__card .has-small-font-size:first-child),
+:where(.ucsc__card .wp-block-image + .has-small-font-size),
+:where(.ucsc__card .wp-block-post-featured-image + .has-small-font-size),
+:where(.ucsc__card .wp-block-heading),
+:where(.ucsc__card .wp-block-post-title) {
 
-		a {
-			text-decoration-color: var(--wp--preset--color--black);
+	a {
+		color: var(--wp--preset--color--ucsc-secondary-blue);
+		text-decoration-line: none;
 
-			&:hover,
-			&:focus {
-				color: var(--wp--preset--color--ucsc-primary-blue);
-				text-decoration-color: var(--wp--preset--color--ucsc-primary-blue);
-			}
+		&:hover,
+		&:focus {
+			color: var(--wp--preset--color--ucsc-primary-blue);
+			text-decoration-line: underline;
 		}
 	}
-
-	.ucsc__card-grid & {
-		height: 100%;
-	}
 }
 
-.ucsc__card-inner {
-	height: 100%;
+/* CASE: Card title follows an overline */
+:where(.ucsc__card .has-small-font-size) {
+
+	+ .wp-block-heading,
+	+ .wp-block-post-title {
+		margin-block-start: -16px !important;
+	}
 }
 
+/* Card CTAs */
```

Comment:
let's remove this if it's not in use.

Reply:
There's one little CASE inside there that sets some special CTA styles when inside of a card grid, so it is in use.